### ISSUE TITLE
Wooden Irrigation Channels not breaking

### ISSUE THREAD

**Issue by FluffyAndFluffier** [InfinityRaider/AgriCraft#753]
On a multiplayer world, I tried to break it with an axe and by hand, but the wooden irrigation channels (specifically Biomes O' Plenty's Redwood) won't break.  I then went into creative, and finally broke them, but their collision models were still there, and they appeared for every other player like they weren't broken.  When I relogged, they appeared in the same place for me.

![2016-12-19_08 28 17](https://cloud.githubusercontent.com/assets/13632484/21316161/25ac51aa-c5c5-11e6-94e3-bab8b715445b.png)

**Comment by InfinityRaider** [comment]
Seems like a serious desync bug, thanks for reporting.

**Comment by InfinityRaider** [comment]
Does this only happen with BoP wood?

**Comment by FluffyAndFluffier** [comment]
I tested Hellbark, Willow, and Pine (both from BoP) Irrigation Channels, and the same still occurred.  They are unable to be broken.  I then tested vanilla Oak and Spruce Irrigation Channels, and the same occurred.  It appears none of the channels are able to be broken.

![2016-12-26_07 48 11](https://cloud.githubusercontent.com/assets/13632484/21482899/b9ab9350-cb3f-11e6-87c1-482fc5a5ed8b.png)
